src/frontend/apps/e2e/__tests__/app-desk/teams-panel.spec.ts

```diff
@@ -35,18 +35,18 @@ test.describe('Teams', () => {
     ).toBeVisible();
   });
 
-  test('002 - check sort button', async ({ page, browserName }) => {
+  test('002 - checks the sort button', async ({ page, browserName }) => {
     const panel = page.getByLabel('Teams panel').first();
 
-    await page.getByRole('button', { name: 'Add a team' }).click();
+    await panel.getByRole('button', { name: 'Add a team' }).click();
 
     const randomTeams = Array.from({ length: 3 }, () => {
       return `team-sort-${browserName}-${Math.floor(Math.random() * 1000)}`;
     });
 
     for (let i = 0; i < 3; i++) {
       await page.getByText('Team name').fill(`${randomTeams[i]}-${i}`);
-      await page.getByRole('button', { name: 'Create a team' }).click();
+      await page.getByRole('button', { name: 'Create the team' }).click();
```

Comment:
I think it should be `Create a team`

Reply:
Could we stick with the design ? I am totally ok with `Create a team`, but if we have to rediscuss all the naming of the elements of the design after integration we will loose lot of time, because we have to change it in the frontend, in the tests e2e (be sure to have it changed correctly everywhere otherwise the tests will failed), then in Crowdin, then import again from Crowdin, until 1 day someone say that the button is not named as it is in the design and change it again.

![image](https://github.com/numerique-gouv/people/assets/25994652/7765ef74-450c-42f0-b61d-c929b15ec415)

Reply:
ok